.github/ISSUE_TEMPLATE/new_product_suggestion.md

```diff
@@ -0,0 +1,26 @@
+---
+name: New Product Suggestion
```

Comment:
```suggestion
name: New product suggestion
```

Reply:
The current each Word starts with capital looks much better because this will be like a Button on issue page.
I suggest we apply with current form and check how it will shown in production then decide to change or keep it

Reply:
All the other buttons are in "Sentence case" :)

![image](https://user-images.githubusercontent.com/1324225/140109891-78591a0b-fadf-4e35-91f3-8ba062173931.png)

![image](https://user-images.githubusercontent.com/1324225/140108821-207aac2d-aab4-4452-a936-3fb646b1e946.png)

![image](https://user-images.githubusercontent.com/1324225/140109376-152dc6a0-e7da-419f-8401-fa33e4268f9a.png)

![image](https://user-images.githubusercontent.com/1324225/140109505-6b222a58-a760-4d1e-aac0-cc1753355982.png)

![image](https://user-images.githubusercontent.com/1324225/140110784-cb09eb99-9c30-44c7-8faa-b303482bdc62.png)

Reply:
done